backend/app/services/diagnosis_agent.py

```
@@ -302,166 +626,192 @@ async def generate_diagnosis(state: DiagnosisState) -> dict:
+            raw_response = ""
+            try:
+                async for chunk in chain.astream({"json_text": json_text}):
+                    raw_response += chunk
+                break
+            except Exception as e:
+                if "RemoteProtocolError" in str(type(e).__name__) or "ConnectTimeout" in str(type(e).__name__):
+                    print(f"LLM streaming dropped connection. Retrying... ({attempt + 1}/{max_retries})")
+                    if attempt < max_retries - 1:
+                        await asyncio.sleep(2)
+                        continue
+                raise e
```

Ruff (0.15.10)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)